src/Utils.tsx

```diff
@@ -0,0 +1,25 @@
+import { Fraction } from 'api';
+
+export function gradeToString(grade: number | Fraction | undefined) {
+	if (grade == undefined) {
+		return '';
+	} else if (typeof grade === 'object') {
+		return `${grade.numerator}/${grade.denominator}`;
+	}
+	return `${grade}`;
+}
+
+export function parseGrade(s: string) {
```

Comment:
I think it might be helpful to import the grade type and specify the return type for this function.

```suggestion
export function parseGrade(s: string): Grade | undefined {
```

This allows people to see that this function returns a grade (rather than number | Fraction | undefined etc.) and it will always be tied to the Grade type